Description: Don't build with the threaded runtime. It causes build failures
  and has a performance penalty.
Origin: upstream, commit:a5db4246656afae3541936269e727d9a9cc39795
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=682433

Index: b/pandoc.cabal
===================================================================
--- a/pandoc.cabal
+++ b/pandoc.cabal
@@ -344,12 +344,12 @@
   else
     Build-depends: base >= 3 && < 4
   if impl(ghc >= 7.0.1)
-    Ghc-Options:   -O2 -rtsopts -threaded -Wall -fno-warn-unused-do-bind -dno-debug-output
+    Ghc-Options:   -O2 -rtsopts -Wall -fno-warn-unused-do-bind -dno-debug-output
   else
     if impl(ghc >= 6.12)
-      Ghc-Options:   -O2 -Wall -threaded -fno-warn-unused-do-bind -dno-debug-output
+      Ghc-Options:   -O2 -Wall -fno-warn-unused-do-bind -dno-debug-output
     else
-      Ghc-Options:   -O2 -Wall -threaded
+      Ghc-Options:   -O2 -Wall
   if impl(ghc >= 7.0.1)
     Ghc-Prof-Options: -auto-all -caf-all -rtsopts
   else
@@ -411,12 +411,12 @@
   else
     Build-depends: base >= 3 && < 4
   if impl(ghc >= 7.0.1)
-    Ghc-Options:   -O2 -rtsopts -threaded -Wall -fno-warn-unused-do-bind -dno-debug-output
+    Ghc-Options:   -O2 -rtsopts -Wall -fno-warn-unused-do-bind -dno-debug-output
   else
     if impl(ghc >= 6.12)
-      Ghc-Options:   -O2 -Wall -threaded -fno-warn-unused-do-bind -dno-debug-output
+      Ghc-Options:   -O2 -Wall -fno-warn-unused-do-bind -dno-debug-output
     else
-      Ghc-Options:   -O2 -Wall -threaded
+      Ghc-Options:   -O2 -Wall
   if impl(ghc >= 7.0.1)
     Ghc-Prof-Options: -auto-all -caf-all -rtsopts
   else
